app/src/main/java/org/httpsknighthacks/knighthacksandroid/MainActivity.java

```diff
@@ -48,10 +59,39 @@ protected void onCreate(Bundle savedInstanceState) {
             }
         }
 
+        if (android.os.Build.VERSION.SDK_INT >= android.os.Build.VERSION_CODES.O) {
+            NotificationChannel channel =
+                    new NotificationChannel("MyNotifications", "MyNotifications", NotificationManager.IMPORTANCE_DEFAULT);
+
+            NotificationManager manager = getSystemService(NotificationManager.class);
+            manager.createNotificationChannel(channel);
+        }
+
         setContentView(R.layout.activity_main);
 
         mRequestQueueSingleton = new RequestQueueSingleton(getApplicationContext());
         getImageAndTitles();
+
+        // Shows the Token to the screen
+
+        /*FirebaseInstanceId.getInstance().getInstanceId()
+                .addOnCompleteListener(new OnCompleteListener<InstanceIdResult>() {
+                    @Override
+                    public void onComplete(@NonNull Task<InstanceIdResult> task) {
+                        if (!task.isSuccessful()) {
+                            Log.w("MainActivity", "getInstanceId failed", task.getException());
+                            return;
+                        }
+
+                        // Get new Instance ID token
+                        String token = task.getResult().getToken();
+
+                        // Log and toast
+                        String msg = getString(R.string.msg_token_fmt, token);
+                        Log.d("MainActivity", msg);
+                        Toast.makeText(MainActivity.this, msg, Toast.LENGTH_SHORT).show();
+                    }
+                });*/
```

Comment:
I'm not super familiar with this codebase but are the commented out parts necessary?

Reply:
Hi, I will fix the commented out code and also thanks for the advice.